### PR TITLE
Set default rollbackBuffers to 0 to improve memory performance

### PR DESCRIPTION
**Pull request by chick**
- Changed default value to 0
- Added another feature to Regression that demonstrates above change
- Added small test to answer question about sign extension of SInt combined with And

### CODE REVIEW

**Comment by chick** [comment]
This fixes a problem with performance reported in Issue #85

File: src/test/scala/treadle/primops/AndOrXor.scala [hunk 1]
```diff
@@ -188,4 +188,25 @@ class AndOrXor extends FreeSpec with Matchers {
       AndInts(() => a, () => b, bitWidth).apply() should be(expected)
     }
   }
+
+  "(7.U.asSInt & true.B.asSInt).asUInt => 7.U" in {
```
Comment (ucbjrl) [inline src/test/scala/treadle/primops/AndOrXor.scala]: Should this be a separate PR?

Comment (chick) [reply src/test/scala/treadle/primops/AndOrXor.scala]: yes, removed it!
